substrait_consumer/functional/common.py

```diff
@@ -200,8 +202,11 @@ def substrait_consumer_function_test(
         actual_result = consumer.run_substrait_query(substrait_plan)
         actual_result = actual_result.rename_columns(
             list(map(str.lower, actual_result.column_names))
-        ).columns[0].data
-        str_result = '\n'.join(map(str, actual_result))
+        ).columns
+        result_list = []
+        for column in actual_result:
```

Comment:
Could we add a blank line between columns?  That'll make it possible to tell the unlikely difference between an item shifting from the end of one column to the front of the next.